### ISSUE TITLE
Tiled map tiles objects support

### ISSUE THREAD

**Issue by BlackDizel** [libgdx/libgdx#4542]
Tiled tiles collision objects support request
http://doc.mapeditor.org/reference/tmx-changelog/#tiled-010 
LibGDX 1.9.5

*.tmx file fragment sample with emded tileset based on image collestion:
```
<tileset>
  <tile id="14">
   <image/>
   <objectgroup draworder="index">
    <object id="1" x="0" y="0" width="256" height="210"/>  <!--rectangle-->
   </objectgroup>
  </tile>
</tileset>
```

*.tsx file fragment sample to tilemap with tileset in source based on tileset image:
```
<tileset>
 <image source="..."/>
 <tile id="34">
  <objectgroup draworder="index">
   <object id="1" x="25" y="34" width="102" height="26"/> <!--rectangle-->
   <object id="2" x="35" y="105" width="95" height="27">
    <ellipse/>
   </object>
   <object id="3" x="146" y="165">
    <polygon points="37,-42 58,-41 82,35 -35,35 -65,0 9,-39 49,-83"/>
   </object>
   <object id="4" x="53" y="19">
    <polyline points="0,0 44,25 -13,71 49,103 -15,141 -8,192"/>
   </object>
  </objectgroup>
 </tile>
</tileset>
```
As I can see, need to update com.badlogic.gdx.maps.tiled.TmxMapLoader#loadTileSet method 
Object parsing exist in com.badlogic.gdx.maps.tiled.BaseTmxMapLoader#loadObject method

**Comment by MobiDevelop** [comment]
I believe there is a PR open for this, which I haven't had a chance to really look at. Will try to get to it sometime soon.

**Comment by j-hiden** [comment]
I think the PR is #3751. Issues #3883 & #4474 also look to be similar this.

**Comment by piotr-j** [comment]
It seems this was merged with #3751, perhaps this issues can be closed?
